client/payment-details/dispute-details/dispute-resolution-footer.tsx

```diff
@@ -273,6 +267,151 @@ const DisputeLostFooter: React.FC< {
 	);
 };
 
+const InquiryUnderReviewFooter: React.FC< {
+	dispute: Dispute;
```

Comment:
Could potentially expand this out to the exact sub-fields needed (if we're not using all fields). Noted this because of `dispute?.id` check later on – we should be able to enforce required data.

Reply:
Although the `dispute?.id` check was resolved in comment https://github.com/Automattic/woocommerce-payments/pull/7242#discussion_r1335267440 as redundant, I've enforced required Dispute properties in 627c158 using `Pick`:

```tsx
dispute: Pick<
	Dispute,
	'id' | 'metadata' | 'status' | 'balance_transactions'
>
```

I could go further and explicitly destructure the individual props, e.g.:

```diff
- dispute={ dispute }
+ id={dispute.id} status={dispute.status}
```

But I think the `Pick` statements sufficiently communicate the required props.

Reply:
Nice! TIL [Pick](https://www.typescriptlang.org/docs/handbook/utility-types.html#picktype-keys) 🙌

Reply:
`Pick` also gets a mention in the [WooPayments TS Guidelines](https://github.com/Automattic/woocommerce-payments/blob/develop/docs/typescript/utility-and-manipulation.md#pick-type---picks-a-set-of-properties). 🙌